SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Rook](https://github.com/rook/rook)
+* [Containerd](https://github.com/containerd/project)
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+and:
+    - Navigate to the [security tab](https://github.com/crossplane/crossplane/security) on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `security@crossplane.io` detailing the issue and steps
+to reproduce.
+
+The reporter(s) can typically expect a response within 24 hours acknowledging
+the issue was received. If a response is not received within 24 hours, please
+reach out to any maintainer directly to confirm receipt of the issue.
```

Comment:
we can link "maintainer" here to https://github.com/crossplane/crossplane/blob/master/OWNERS.md#maintainers to make it more discoverable :)